lib/cocoapods/installer/user_project_integrator.rb

```diff
@@ -41,18 +37,24 @@ class UserProjectIntegrator
       #
       attr_reader :targets
 
+      # @return [Boolean] whether to use input/output paths for build phase scripts
+      #
+      attr_reader :use_input_output_paths
+      alias use_input_output_paths? use_input_output_paths
+
       # Init a new UserProjectIntegrator
       #
       # @param  [Podfile]  podfile @see #podfile
       # @param  [Sandbox]  sandbox @see #sandbox
       # @param  [Pathname] installation_root @see #installation_root
       # @param  [Array<AggregateTarget>] targets @see #targets
       #
-      def initialize(podfile, sandbox, installation_root, targets)
+      def initialize(podfile, sandbox, installation_root, targets, use_input_output_paths)
```

Comment:
maybe let's start using kwargs for option-like parameters like these?

Reply:
sounds good to me. I know CocoaPods doesn't really use kwargs much so I didn't want to break existing norms but that might make more sense